demo/external-logging-elasticsearch/packager-config.yml

```diff
@@ -1,28 +1,28 @@
 bundle:
   groupId: "io.jenkins.tools.war-packager.demo"
-  artifactId: "external-task-logging-logstash"
+  artifactId: "jenkins-external-task-logging-elk-demo"
   vendor: "Jenkins project"
   title: "Jenkins External Task Logging demo for Elasticsearch"
   description: "Jenkins External Task Logging demo for Elasticsearch, packaged as a single WAR file. It automatically configures logging on startup"
 buildSettings:
   docker:
-    base: "jenkins/jenkins:2.107.3"
-    tag: "jenkins/demo-external-task-logging-logstash"
+    base: "jenkins/jenkins:2.121.1"
+    tag: "jenkins/demo-external-task-logging-elk"
     build: true
 war:
   groupId: "org.jenkins-ci.main"
   artifactId: "jenkins-war"
   source:
-    version: 2.107.3
+    version: 2.121.1
 plugins:
   - groupId: "org.jenkins-ci.plugins"
     artifactId: "structs"
     source:
-      version: 1.10
+      version: 1.14
```

Comment:
Sad—this seemed easier to manage from a POM.

Reply:
Vote for https://issues.jenkins-ci.org/browse/JENKINS-51574